plasmapy/atomic/tests/test_particle_class.py

```diff
@@ -196,6 +203,7 @@
         'particle': 'H-1 0+',
         'element': 'H',
         'isotope': 'H-1',
+        'isotope_name': 'hydrogen-1',
```

Comment:
There are two names that we could adopt for [H-1](https://en.wikipedia.org/wiki/Isotopes_of_hydrogen#Hydrogen-1_(protium)): `'hydrogen-1'` and `'protium'`.  Do we have a preference between the two?  Using `'protium'` would be more consistent with the way we use `'deuterium'` and `'tritium'`, but `'protium'` is less well known.

Reply:
I like hydrogen-1 here, I don't think I've ever heard "protium" used. That's not to say I've heard much.